### DISCUSSION TITLE
yolov5 tf2-keras implement.

### DISCUSSION THREAD

**Discussion by yyccR** [ultralytics/yolov5#7115] [Show and tell]
open source:
```
https://github.com/yyccR/yolov5_in_tf2_keras
```

training result:
![image](https://user-images.githubusercontent.com/14156500/159614463-2712c0ef-32db-40da-be10-fa1e7e6fba40.png)

<img width="709" alt="image" src="https://user-images.githubusercontent.com/14156500/159614482-6ff9ad85-aff3-4d64-b5fb-a95eeb731cc6.png">

**Comment by glenn-jocher** [comment]
@yyccR awesome, nice contribution!! Results look great!

**Comment by samanAntoni** [comment]
@yyccR I am reviewing your code, but unfortunately in many cases the comments are not readable for me. Do you also have an English version from your code(comments)?

**Reply by yyccR** [reply]
Sorry bro, I think you can try to learn a little Chinese and then you will be interested in her 5000 years of history and culture.  ^_^
